Formula/swift-format@5.2.rb

```diff
@@ -0,0 +1,28 @@
+class SwiftFormatAT52 < Formula
```

Comment:
This shouldn't be added unless there's sufficient demand and ongoing support from Apple.

Reply:
Reverted.